src/server/handlers/launchpad.js

```diff
@@ -653,6 +653,12 @@ export const authorizeSnap = async (req, res) => {
   }
 };
 
+export async function internalGetSnapBuilds(snap, start = 0, size = 10) {
```

Comment:
Do the parameter defaults make the default values on lines 665 and 666 redundant?